app-backend/batch/src/main/scala/com/azavea/rf/batch/aoi/FindAOIProjects.scala

```diff
@@ -43,7 +43,9 @@ case class FindAOIProjects(implicit val xa: Transactor[IO]) extends Job {
         .to[List]
     }
 
-    aoiProjectsToUpdate.transact(xa).unsafeRunSync.foreach(println)
+    aoiProjectsToUpdate.transact(xa).unsafeRunSync.foreach(
+      (projectId: UUID) => println(s"Project to update: ${projectId}")
```

Comment:
I think it would be good to add a comment here to help prevent this from happening again (or at least make it more difficult)